mobify/platform-scaffold
# [Integration manager] demandware cart commands

This PR adds commands for the cart page to the demandware connector, it also adds product images to the cart data in the store. 

**Note:** This PR does not include work for the wishlist "save for later" functionality because it requires setting up the ability to transfer a cart on login, so I'll open another PR for that functionality. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1271
 **Linked PRs**: n/a

## Changes
- Added command to remove item from the cart
- Added command to update an item quantity in the cart
- Added command to "fetch" the page content
- Updated the getCart command to also fetch product images (only fetches an image if it's not in the store) 

## How to test-drive this PR
- Run `npm run dev`
- Preview [the demandware site](https://preview.mobify.com/?url=https%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2Fdefault%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to your cart
- navigate to the cart page and confirm the page is working as expected
